x-pack/plugins/cloud_security_posture/public/pages/rules/rules_container.tsx

```diff
@@ -191,24 +194,16 @@ export const RulesContainer = () => {
           }}
           setSelectedRuleId={setSelectedRuleId}
           selectedRuleId={selectedRuleId}
-          refetchStatus={rulesStates.refetch}
+          refetchRulesStatus={rulesStates.refetch}
           selectedRules={selectedRules}
           setSelectedRules={setSelectedRules}
         />
       </EuiPanel>
       {selectedRuleId && (
         <RuleFlyout
-          rule={{
-            ...{
-              status:
-                filteredRulesStates.find((e) => e.rule_id === selectedRuleId)?.muted === true
-                  ? 'muted'
-                  : 'unmuted',
-            },
-            ...{ metadata: rulesPageData.rules_map.get(selectedRuleId!)?.metadata! },
-          }}
+          rule={rulesFlyoutData}
           onClose={() => setSelectedRuleId(null)}
-          refetchStatus={rulesStates.refetch}
+          refetchRulesStatus={rulesStates.refetch}
```

Comment:
i noticed that we use 2 terminologies
1. rules status
2. rules states

we need to stick to 1, it can lead to annoying problems later on.. i think the api is rules states so lets stick to those